doc/changelog/09-coqide/14863-coqide-opt-unicode-binding.rst

```diff
@@ -0,0 +1,6 @@
+- **Fixed:**
+  It is now possible to deactivate the unicode completion
+  mechanism
+  (`#14863 <https://github.com/coq/coq/pull/14863>`_,
+  fixes `#14856 <https://github.com/coq/coq/issues/14856>`_,
```

Comment:
As is, this isn't sufficient to consider the issue fixed, so this mention should be removed or amended. Also, the way to disable the Unicode completion mechanism should be documented, both in the changelog, and in the documentation of the feature.

Reply:
Done.